### DISCUSSION TITLE
How do I add the social media icons to the sidebar, I still don't get it

### DISCUSSION THREAD

**Discussion by HongTEM** [Godiesc/firefox-gx#156] [General]
I downloaded the extensions but don't know how to put it into the sidebar, is it something wrong with my method?

**Comment by HongTEM** [comment]
Nvm, found out you need to pin it before you can add it to the toolbar

**Comment by godcanonen** [comment]
Could you explain pls how to pin them?

**Reply by Godiesc** [reply]
Hi,
first, you pin the extensions from the 'extensions button' to the toolbar.

![image](https://github.com/user-attachments/assets/32fb80d4-5634-49c8-aa58-509dd5251623)

after that you can move that buttons to the left-sidebar using the [customization mode](https://support.mozilla.org/en-US/kb/customize-firefox-controls-buttons-and-toolbars)
